### PR TITLE
 B-25966  uses operator-sdk to build docker image

### PR DESCRIPTION
**Pull request by drewwells**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -26,6 +31,15 @@ operator-chart:
 
 deploy: .id deploy/cluster.yaml generate operator-chart operator-chart operator-todo
 
+image: .image-$(GIT_COMMIT)
+
+push: image
+	docker push $(REGISTRY)/$(IMAGE_REPO):$(GIT_COMMIT)
+
+.image-$(GIT_COMMIT):
+	OPERATOR_NAME=$(OPERATOR_NAME) .bin/operator-sdk-$(OPERATOR_SDK_VERSION) build $(REGISTRY)/$(IMAGE_REPO):$(GIT_COMMIT) --verbose
```
Comment (hryan-ib) [inline makefile:40]: Will this pack all that we need into the image? Using the kops binary approach, I have to install AWS CLI and the kops binary onto the container for it to work. I am not sure if this approach needs a Dockerfile to go along with it that will add what we need.

Comment (drewwells) [reply makefile:40]: No it's just the first step of creating the binary. I'll make more PRs for the other steps of container creation process

Comment (hryan-ib) [reply makefile:40]: 👍 I have been working on the docker file, chart and binary approach in this branch https://github.com/infobloxopen/cluster-operator/tree/container

Comment (drewwells) [reply makefile:40]: Will you open a draft PR for it

**Comment by drewwells** [comment]
@hryan-ib is already working on this in a branch

**Pull request closed without merging**